File: Be/interact-service/src/main/java/com/interact/dto/response/FeedBackResponse.java
package com.interact.dto.response;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class FeedBackResponse {
    String id;

    int rating;
    String userId;
    String feedBack;
    Boolean isAvailable;
    String imgAvatar;
    String userName;
    String itemId;
    String feedBackType;
}

File: Fe/client/src/dto/response/FeedBackResponse.ts
import { listResponse } from "@/dto/response/listResponse";

export type FeedBackResponse = {
	id: string
	rating: number;
	userId: string;
	userName: string;
	feedBack: string
	imgAvatar: string
	isAvailable: boolean
	itemId: string
	feedBackType: string
}

export type FeedBackResponses =listResponse<FeedBackResponse> & {
	result: {
		averageRating: number;
	}
}